libraries/3rdparty/tracealyzer_recorder/streamports/TCPIP/trcStreamingPort.c

```diff
@@ -53,8 +53,7 @@
 /* TCP/IP includes */
 #include "lwip/tcpip.h"
 #include "lwip/sockets.h"
-
-int errno;
+#include "lwip/errno.h"
```

Comment:
The same comment about lwip above applies here as well; I have emailed percepio support to let them know about this change.  If they accept it, we will incorporate their change here upon receiving the updated version.

Reply:
I've heard back from Percepio – they will be incorporating this change into their next release of the tracealyzer recorder library.  When it becomes available, we'll integrate it into this repository.

Reply:
Here's the comment from Percepio - _"The fix has been checked in on our side and will be included in Tracealyzer v4.3.6....make the change in your version and change the version number of that specific file to 'v4.3.5.1'"_